E2a-TRAS/GL01/README.md

```diff
@@ -0,0 +1,137 @@
+
+
+
+
+## Validation criteria
+:zap: For this test case to be considered passed **all capabilities listed in this section**, and **the ones of pre-required tests** shall be verified, with no exception. :zap:
+
+### General
+
+- All the concept templates must be correctly implemented as presented in the validation criteria
+- At least 1 instance of each entity listed in [Itemised Roots](#Itemised-Roots) is present in the file
+- All validation criteria of the pre-required tests shall apply here too
+
+
+### Global Positioning
+
+> **Acceptance criteria**
+ For the **Global Positioning** capability, the validation procedure must verify that:
+> - `IfcMapConversion` entity is used to transform the local engineering coordinate system, often called world coordinate system (WCS), into the coordinate reference system of the underlying map.
+> - `IfcProjectedCRS` entity is used for representing the coordinate reference system of the map to which the map translation of the local engineering coordinate system of the engineering project relates.
+>
+> If present, all criteria listed in [Usages, Constraints & Logic](#Usages,-Constraints-&-Logic), and in the same section of precondition tests, shall be verified too.
+
+
+| **ID**  | **CRITERIA**                                      | **VALUE** | **COMMENT** |
+|---------|---------------------------------------------------|-----------|-------------|
+| ENAT_01 | Requested entities (and attributes) exist in file | See below |             |
+
+<details><summary>Entities and attributes</summary>
+
+`IfcMapConversion`
+
+| #  | Attribute        | Value / Instructions                          |
+|----|------------------|-----------------------------------------------|
+| 1  | SourceCRS        | Points to `IfcGeometricRepresentationContext` |
+| 2  | TargetCRS        | Points to `IfcProjectedCRS` (see below)       |
+| 3  | Eastings         | 0                                             |
+| 4  | Northings        | 0                                             |
+| 5  | OrthogonalHeight | 0                                             |
+| 6  | XAxisAbscissa    | 1                                             |
+| 7  | XAxisOrdinate    | 0                                             |
+| 8  | Scale            | 1                                             |
+| 9  | ScaleY           | 1                                             |
+| 10 | ScaleZ           | 1                                             |
+
+`IfcProjectedCRS`
+
+| # | Attribute     | Value / Instructions                        |
+|---|---------------|---------------------------------------------|
+| 1 | Name          | 'EPSG:3065'                                 |
+| 2 | Description   | 'Istituto Geografico Militare 1995 (IGM95)' |
+| 3 | GeodeticDatum | 'EPSG:3065'                                 |
+| 4 | VerticalDatum |  $                                          |
```

Comment:
What would be the correct VerticalDatum here? Or is the whole model to be interpreted without heights?

I'd suppose that the heights are also defined in some vertical CRS. Please confirm with data source.

Reply:
Vertical Datum added